sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -223,6 +223,8 @@ class Analyzer(
           case other => Alias(other, other.toString)()
         }
 
+        val attributeMap = groupByAliases.map(a => (a -> a.toAttribute.withNullability(true))).toMap
```

Comment:
Friendly ping, if we can fix this last comment we can probably include this in 1.6-RC1.